templates/data-provider/claim-card.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Card, Button } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.module.css'
+
+const ClaimCard = ({ name }) => (
+  <Card className={classNames.use(styles.card, styles.claimCard)}>
```

Comment:
If you use a card, it should be either a `<section>` or an `<aside>` section (I am not sure which).

However, it's designed in a way that this is a footer to `<aside>` which already contains sections and it should not have a spacing arount.

Reply:
I was expecting that these sidebar cards can be positioned on different places on mobile view, i.e. claim card after two outputs results and so on hence I didn't consider it as a footer. 


I would say the whole sidebar should be aside and the inner card can stay `div`. 🤔

Reply:
According to the [spec](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element), aside can contain nested sections (check what "[Flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2)" means).

Reply:
ok why not thanks :)